src/pages/BasePage.java

```diff
@@ -3,42 +3,38 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-import java.util.Collection;
-
 public class BasePage {
     public WebDriver driver;
     public WebDriverWait wait;
```

Comment:
Variables should be private or protected.

Reply:
@vladimirKorzhukov
Please read my notes! I left a bunch of comments and a message in the txt file.